FunctionalTableData/CellStyle.swift

```diff
@@ -39,15 +39,16 @@ public struct CellStyle {
 	public var selectionColor: UIColor? = CellStyle.selectionColor
 	/// The view's background color.
 	public var backgroundColor: UIColor?
-	/// The view that is displayed behind the cell’s other content.
-	public var backgroundView: UIView?
```

Comment:
This is a breaking API change since we're deleting a public property. The fix is pretty trivial. Should we keep the property and maintain backward compatibility, or just break clients and increment the major version number?
@raulriera @g-Off

Reply:
We should keep the property, its already using a second initializer. We can just deprecate this property and fix all the usages. That is what we have been doing lately

Reply:
Great, i'll make this change